### PR TITLE
Add a track and cohort filter to CSV export

### PR DESCRIPTION
**Pull request by davestgermain**
For [EDUCATOR-4294](https://openedx.atlassian.net/browse/EDUCATOR-4294)

Requires https://github.com/edx/edx-platform/pull/20677

### CODE REVIEW

File: staff_graded/static/html/staff_graded.html [hunk 1]
```diff
@@ -44,8 +59,6 @@ <h3 class="hd hd-3 problem-header">{{display_name}}</h4>
         </div>
     </div>
 </div>
-<script type="text/javascript">
-</script>
```
Comment (matthugs) [inline staff_graded/static/html/staff_graded.html]: thx

File: staff_graded/static/html/staff_graded.html [hunk 2]
```diff
@@ -16,7 +16,22 @@ <h3 class="hd hd-3 problem-header">{{display_name}}</h4>
         <ul>
             <li><strong>{% trans "Step 1:" %}</strong> {% trans "Export scores." %}<br>
                 <p>{% trans "Download a grading CSV and use this as a template to assign scores for this problem." %}</p>
-                <p><a href="{{export_url}}" class="btn btn-brand">{% trans "export scores" %}</a></p>
+                <p>
+                    {% trans "Choose a track and/or cohort:" %}
+                    <select class="track-field">
+                        <option value="">{% trans "(All Tracks)" %}</option>
+                        {% for track, track_name in available_tracks %}
+                        <option value="{{track}}">{{track_name}}</option>
```
Comment (matthugs) [inline staff_graded/static/html/staff_graded.html]: Could you indent again from here? (and for the other loop, please)

File: staff_graded/staff_graded.py [hunk 3]
```diff
@@ -93,6 +103,16 @@ def student_view(self, context=None):
         context['display_name'] = self.display_name
         context['is_staff'] = self.runtime.user_is_staff
 
+        course_id = self.location.course_key
+        context['available_cohorts'] = [cohort.name for cohort in get_course_cohorts(course_id=course_id)]
+        if CourseMode:
+            context['available_tracks'] = [
+                (mode.slug, mode.name) for mode in
+                CourseMode.modes_for_course(course_id, only_selectable=False)
+                ]
```
Comment (matthugs) [inline staff_graded/staff_graded.py]: oof, feels like we should set up pylint for this sort of indentation weirdness

Comment (matthugs) [reply staff_graded/staff_graded.py]: @davestgermain did you set up pylint in your updates to this branch? Shall I enter a ticket for adding it to this repo?

Comment (davestgermain) [reply staff_graded/staff_graded.py]: Yes. Pylint had no problem with those lines.

**Pull request merged**